Safety/Safety-Systems/AuxTel-Safety-Gate-Procedures.rst

```diff
@@ -0,0 +1,198 @@
+
+1. Press the :guilabel:`Release` button on the gate. 
+    The blue *Lock/Area Safe* indicator will turn off.
+
+2. Move the **handle** and physically move the gate to the open position.
+
+    .. Fig6. Lock button                                                
+
+    .. figure:: ./_static/lock-release-buttons.png
+        :align: left
+        :width: 250px
+        
+        :guilabel:`Release` and :guilabel:`Lock` buttons
+
+
+.. _AuxTel-Safety-Gate-Procedures-Gate-operation-Closing-the-gate:
+
+Closing the gate
+------------------
+.. section 3
+
+When the gate is fully open, to close it:
+
+1. Move the gate to the **closed position**.
+
+2. Insert the handle into the frame. 
+    Notice that the *Safety Gate* indicator lights up.
+
+    .. figure:: ./_static/handle-in-closed-position.png
+      :width: 250px
+      
+      Handle in closed position
+
+    .. Fig5. Handle in closed position
+
+3. Press the :guilabel:`Lock` button. 
+    You should hear a click, and the *Lock/Area Safe* indicator should illuminate in blue, signaling that the handle cannot be moved and the gate is securely closed. 
+
+    .. Fig7. Locked gate indicator
+
+    .. figure:: ./_static/locked-gate-indicator.png
+        :width: 250px
+        
+        Locked gate indicator
+
+.. _AuxTel-Safety-Gate-Procedures-Verify-Telescope-and-Dome-Drive-Interlocks:
+
+Verify Telescope and Dome Drive Interlocks 
+===========================================
+
+.. section 8
+
+To determine if the drives have a **triggered interlock**, due to the gate open, gate in Error, E-STOP action, or general failure. 
+Do the following:
+
+* Check the **Pilz controller** inside the AuxTel Main Cabinet.
+
+* According to the **lights** if they are:
+    * Off, an interlock is triggered.
+    * Illuminated, the system is functioning correctly.
+
+.. Fig 11. Pilz controller status
+
+.. figure:: ./_static/pilz-controller-status.png
+      :width: 250px
+      
+      Pilz controller status
+
+
+
+.. _AuxTel-Safety-Gate-Procedures-Enable-the-gate-after-emergency-release:
+
+Enable the gate after emergency release
+========================================
+
+.. section 5
+
+1. **Rearm the red handle** and move it to its normal position, as shown in figure below. 
+
+2. Press the :guilabel:`Release` button for 5 seconds. 
+    The Error mode will be reset, indicated by the red light disappearing.
+
+.. Fig8 . Emergency release mechanism
+
+.. figure:: ./_static/emergency-release-mechanism.png
+      :width: 250px
+      
+      Emergency release mechanism
+
+
+
+.. _AuxTel-Safety-Gate-Procedures-Reset-gate-Emergency-Stop:
+
+Reset gate Emergency Stop
+===========================
+
+.. section 6
+
+Press the :guilabel:`Reset` button on one of the AuxTel 1st floor cabinets (Main cabinet or the Dome control cabinet).
+
+.. Fig9. Emergency stop button
+
+.. figure:: ./_static/emergency-stop-button.png
+      :width: 250px
+      
+      Emergency stop button
```

Comment:
This image shows the Emergency stop button on the safety gate but the text is talking about the Reset button in the Main cabinet. I would suggest to add or replace with that picture to avoid confusion.